### PR TITLE
added release notes up to 10.17.0.122

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-17-0/devices-10-17-0.md [hunk 1]
```diff
@@ -58,6 +58,13 @@ In the <b>Filter target devices</b> grid in bulk operations and in the <b>Assign
 <td>10.16.66.0</td>
 </tr>
 
+<tr>
+<td>LWM2M</td>
+<td>The performance of the <code>migrateLwm2mDevices</code> operation has been improved. New command line arguments have been introduced with the operation. A list of legacy LWM2M devices can be directly specified from the shell command. Moreover, the migration of the LWM2M client registration objects can also be skipped using an argument. Refer to <a href="https://cumulocity.com/guides/protocol-integration/lwm2m/#migration-of-the-lwm2m-devices" class="no-ajaxy">LWM2M > LWM2M connector device > Migration of the LWM2M devices</a> in the <i>Protocol integration guide</i> for more details about the usage.</td>
```
Comment (rssag) [inline content/release-10-17-0/devices-10-17-0.md:63]:
```suggestion
<td>The performance of the <code>migrateLwm2mDevices</code> operation has been improved. New command line arguments have been introduced with the operation. A list of legacy LWM2M devices can be directly specified from the shell command. Moreover, the migration of the LWM2M client registration objects can also be skipped using an argument. Refer to <a href="https://cumulocity.com/guides/protocol-integration/lwm2m/#migration-of-the-lwm2m-devices" class="no-ajaxy">LWM2M > LWM2M connector device > Migration of the LWM2M devices</a> in the <i>Protocol integration guide</i> for details.</td>
```

File: content/release-10-17-0/platform-10-17-0.md [hunk 2]
```diff
@@ -137,6 +137,13 @@ Fix Version</th>
 <td>10.16.13.0</td>
 </tr>
 
+<tr>
+<td>Authentication</td>
+<td>Fixed an issue with the device requests counter being increased while switching between the standard applications (Administration, Cockpit, Device management).</td>
```
Comment (rssag) [inline content/release-10-17-0/platform-10-17-0.md:142]: see 10.16 PR - i suggested removing the "s" at the end of "requests" - but not sure how the component is actually called

File: content/release-10-17-0/devices-10-17-0.md [hunk 3]
```diff
@@ -97,9 +104,16 @@ Issue</th>
 <td>10.16.119.0</td>
 </tr>
 
+<tr>
+<td>Connectivity</td>
+<td>Due to a compatibility error of the connectivity microservice with the Kite provider, an error occurred on the metrics page, when the monthly consumption data was an out-of-scope integer. This has been fixed by changing the data type.</td>
```
Comment (BeateRixen) [inline content/release-10-17-0/devices-10-17-0.md:109]:
```suggestion
<td>Due to compatibility issues of the connectivity microservice with the Kite provider, an error occurred on the metrics page, when the monthly consumption data was an out-of-scope integer. This has been fixed by changing the data type.</td>
```

File: content/release-10-17-0/platform-10-17-0.md (hunk 2 above)
Comment (BeateRixen) [inline content/release-10-17-0/platform-10-17-0.md:142]:
```suggestion
<td>Fixed an issue with the device request counter being increased while switching between the standard applications (Administration, Cockpit, Device management).</td>

```

**Pull request merged**